test/util/data/tapyrus-util-test.json

```diff
@@ -733,5 +733,59 @@
     ],
     "check":"check_generateKey",
     "description": "Ensure generatekey is successful in dev mode"
+  },
+  {
+    "exec": "./tapyrus-genesis",
+    "args": [
+      "-time=1563342688",
+      "-signblockpubkey=04473757a955a23f75379820f3071abf5b3343b78eb54e52373d06259ffa6c550bd0acc38ff1b182a5de0c795aa7109e8348ca5436d65a267ef134af595ec6c75e",
+      "-signblockprivatekey=5KbckfYoaBR1a7PuR2JAE2E1jGZ4hMmmCT9nNwJ3rfqf78oWr92"
+    ],
+    "return_code": 1,
+    "error_txt": "Error: Uncompressed Aggregate Public Keys are not supported",
+    "description": "Ensure uncompressed pubkeys are not accepted as aggregate pubkey."
+  },
+  {
+    "exec": "./tapyrus-genesis",
+    "args": [
+      "-dev",
+      "-time=1563342688",
+      "-signblockpubkey=04473757a955a23f75379820f3071abf5b3343b78eb54e52373d06259ffa6c550bd0acc38ff1b182a5de0c795aa7109e8348ca5436d65a267ef134af595ec6c75e",
+      "-signblockprivatekey=L55noj9NZSxyX9E6M2ijiUQSDZeXK8ga5YRmHpeFcERmBnvc7bS3"
+    ],
+    "return_code": 1,
+    "error_txt": "Error: Uncompressed Aggregate Public Keys are not supported",
+    "description": "Ensure uncompressed pubkeys are not accepted as aggregate pubkey in dev mode."
```

Comment:
> Ensure uncompressed pubkeys are not accepted as aggregate pubkey in dev mode.

but,, `L55noj9NZSxyX9E6M2ijiUQSDZeXK8ga5YRmHpeFcERmBnvc7bS3` dose not dev mode format.

Reply:
These keys were generated  from an online tool. sorry, I hadn't cross checked it. 
But the error here is generated from the pub key. Private key is not yet validated when the error happens.

Reply:
> But the error here is generated from the pub key. Private key is not yet validated when the error happens.

I see. However, considering the intent of the test, the key should be in the correct format.